packages/app/src/specs/generators/ExpandableFileChooser.vue

```diff
@@ -0,0 +1,165 @@
+<!--
+/**==============================================
+ * *          ExpandableFileChooser.vue
+ * Filter a list of files by a mix of glob pattern
+ * and a search string that includes a file's relative
+ * path.
+ *
+ * When clicked, the files will expand to display the
+ * #expanded-content slot.
+ *
+ * Features to note: debouncing + loading + row expanding
+ * ==============================================
+ * *                Debouncing
+ *
+ * ? To prevent calling update too frequently we debounce
+ * ? both the "No Results" UI and the update event
+ *
+ * ============================================**/
+-->
+
+<template>
+  <CreateSpecModalBody
+    variant="bare"
+    class="bg-white flex flex-col px-24px
+  relative"
+  >
+    <FileMatch
+      ref="fileMatchRef"
+      v-model:pattern="filePathSearch"
+      v-model:extensionPattern="localExtensionPattern"
+      class="bg-white pt-24px pb-12px top-0px z-10 sticky"
+      :matches="matches"
+    >
+      <template
+        v-if="loading"
+        #matches
+      >
+        <i-cy-loading_x16 class="h-24px mr-10px animate-spin w-24px" />
+      </template>
+    </FileMatch>
+
+    <div
+      v-show="loading"
```

Comment:
Must have missed this before but was there a specific reason to use `v-show` over `v-if` and `v-else` here?

Reply:
There was not - in fact, this doesn't even need to be here